postal/service.go

```diff
@@ -175,13 +179,6 @@ func (s Service) Deliver(dependency Dependency, cnbPath, layerPath, platformPath
 	return nil
 }
 
-// Install will invoke Deliver with a hardcoded value of /platform for the platform path.
-//
-// Deprecated: Use Deliver instead.
```

Comment:
@ryanmoran Is this ok to deprecate now?

Reply:
It is deprecated. Do you mean remove?

Reply:
Oh yes, that's what I meant :)

Reply:
Assuming you're good with this given your comments on the removal of associated tests.